### PR TITLE
Remove navbar to optimize usage of vertical space.

### PR DESCRIPTION
**Pull request by ongspxm**
```
After the removal of the feature to support multiple reports, the navigation
bar contains only the reposense logo.

The bar now plays only an aesthetic role and has no functional purpose, which
is a waste of vertical space.

Similarly the filters on the summary tab could be more concisely displayed to
fully ultilize vertical space.

Let's make the necessary UI changes to improve vertical space usage.
```

image

### CODE REVIEW

**Comment by ongspxm** [comment]
@yong24s @yamidark @AdityaA1998 review then i will tag @eugenepeh 

oopss...

**Comment by yong24s** [comment]
LGTM.

File: template/index.html [hunk 1]
```diff
@@ -1 +1 @@
...
...th in getContributionBars(user.totalCommits)" v-bind:style="{ width: width+'%' }" class="summary-chart__contrib--bar"></div></div></div></div></div></div></vuetemplate><vuetemplate id="v_authorship"><div id="authorship"><div class="title"><div class="repoName">{{ repo }}</div><div class="author">{{ author }}</div></div><div v-if="isLoaded" class="files"><div v-for="file in files" class="file"><div onclick="toggleNext(this)" class="title">{{ file.path }}</div><pre class="hljs java"><code><template v-for="segment in file.segments"><div v-bind:class="{ untouched: !segment.authored }" class="segment"><div v-if="!segment.authored" onclick="toggleNext(this)" class="closer"><span>untouched code (click to toggle)</span></div><div v-if="segment.authored" class="code">{{ segment.lines.join("\n") }}</div><div v-else="v-else" style="display:none;" class="code">{{ segment.lines.join("\n") }}</div></div></template></code></pre></div></div><div v-else="v-else" class="empty">loading...</div></div></vuetemplate><script src="static/js/api.js"></script><script src="static/js/v_summary.js"></script><script src="static/js/v_authorship.js"></script><script src="static/js/main.js"></script></body></html>
```
[the remark this answers is not in the document]
Comment (ongspxm) [reply template/index.html]: eh, this one is generated file leh... from the `index.jade`

**Comment by eugenepeh** [comment]
If possible, yes. It's not good to have generated files in the repository as it creates conflict when multiple people working on the same thing.

**Comment by ongspxm** [comment]
Okay, i can open another pr to add the building of all these files to gradlew

**Comment by eugenepeh** [comment]
image

Perhaps make all the selectable field to be on the left and all the check boxes on the right?
Btw, for the `up/down` boxes circled in red, it seems pretty redundant. Can it be removed?

**Comment by damithc** [comment]
Another thing: week|day should be a drop-down rather than a check box; we'll be adding a third value 'commit' to it in near future.

**Comment by ongspxm** [comment]
> Another thing: week|day should be a drop-down rather than a check box; we'll be adding a third value 'commit' to it in near future.

my interpretation of the week vs day was that for week, you group the commits and display them as one ramp, whereby for day, the commits are grouped per day and displayed as one ramp

for the commits, we are just skipping the "display all commits as one ramp" part and show up as multiple small spikes instead.

I am not sure in such a use case, a dropdown menu will be appropriate

**Comment by damithc** [comment]
> my interpretation of the week vs day was that for week, you group the commits and display them as one ramp, whereby for day, the commits are grouped per day and displayed as one ramp

That is correct.

> I am not sure in such a use case, a dropdown menu will be appropriate

From the viewer's POV, this is a case of 'choose one of three options': show all commits | group commits by day | group commits by week. Choose 1-of-n usually maps to a group of radio buttons or a drop-down; latter is preferred here due to space constraints.

I'm open to other suggestions.

**Comment by ongspxm** [comment]
i am thinking more of a "group commits by week | group commits by day", and a checkbox that "consolidates commits into one ramp" (not the exact name, but its wut it does)

**Comment by damithc** [comment]
> i am thinking more of a "group commits by week | group commits by day", and a checkbox that "consolidates commits into one ramp" (not the exact name, but its wut it does)

A bit too complicated (there is a hidden link between the two settings) and needs more space too. 
How about `Ramp granularity`: `commit` | `day` | `week` ?

> this makes it such that the commit option isn't be part of the time-scale change

Not sure what's the implication here. From what I see, only the ramps should change when the setting is changed; the scale of the timeline should stay the same for all three options.

May be there is some implication I missed. We can discuss tomorrow.

**Comment by ongspxm** [comment]
@eugenepeh ready for merge?

**Comment by eugenepeh** [comment]
Just a question for your message:

After the removal of the feature to support multiple reports, the
navigation bar contains only the reposense logo.

The bar now plays only an aesthetic role and has no functional purpose,
which is a waste of vertical space.

Similarly the filters on the summary tab could be more concisely
displayed to fully utilize the vertical space.

Let's set the default path for retrieving report data to `/`, and make
the necessary UI changes to improve vertical space usage.


`set the default path for retrieving report data to /`, which part of the code are you referring to?

**Comment by ongspxm** [comment]
> Let's set the default path for retrieving report data to /, and make
the necessary UI changes to improve vertical space usage.

dun know how to copy code, so here it is, looks like i have sneaked it in in #198 
https://github.com/reposense/RepoSense/pull/187/files#diff-159d355dfa2dcad838172176372f7b82R5

i have modified the message.

**Pull request merged**